package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "@eth-optimism/contracts": "^0.0.2-alpha.7",
+    "new-contracts": "npm:@eth-optimism/contracts@0.1.4",
```

Comment:
You can't just do an upgrade?

Reply:
Got an error when upgrading and got frustrated trying to upgrade the batch submitter to the newer version. Definitely tech debt and gotta go back and upgrade everything to the latest version 😭

Reply:
I had a similar error and fixed it by deleting the `yarn.lock`. Does that fix it?

Reply:
Just tried and also downgraded to 1.4 and no luck 😕